### ISSUE TITLE
feature request: add visual cue that atom boxes are receiving data

### ISSUE THREAD

**Issue by porres** [pure-data/pure-data#1321]
I'm proposing that atom boxes (both float and symbol) could 'flash' for a moment saying they received data (kind of like in a [bng] object) - but it would be having a thicker border for a brief moment.

We have a thicker border for when the atom box is active already and it goes back to normal when it is not active. My idea would be to make it thicker as it'd be active.

This could be an optional (non default) parameter that you can set inside properties. A check box to tell it to flash, and also maybe a flashing period.

The idea is that people actually need this and end up using a hacky way just to get a visual cue by using [bng]. This is specially useful for documentation too and is related to the discussion in => https://github.com/pure-data/pure-data/issues/1320#issuecomment-846676052

**Comment by umlaeute** [comment]
interesting idea.

it probably should be made optional though.

**Comment by porres** [comment]
> it probably should be made optional though.

yup, not there by default but you could set it up in the properties
